README.md

```diff
@@ -36,6 +36,43 @@ Type the name of [any `iron-*` or `paper-*` element](https://elements.polymer-pr
 </dom-module>
 ```
 
+### [pe2015] polymer element ES2015
+
+```html
+<dom-module id="${2}">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+  </template>
+  <script>
+    (function (Polymer) {
+      'use strict';
+      class ${1} {
+        beforeRegister() {
+          this.is = '${2}';
+
+          this.properties = {
+            ${3}
+          };
+        }
+
+        attached() {
+           this.async(() => {
```

Comment:
Probably confusing more then anything. I wanted to provide a little extra boiler plate then the regular `pe` snippet does. I wanted to cover things like defining properties and adding methods because it's slightly different then in ES5. 

I originally had the `ready` call back there. However, it seems like almost time i've tried to use the `ready` call back what I really wanted was `domReady`. Since this was originally a personal snippet, I changed ready to what you see here. 

I can change it to whatever method you want but the underlying problem is the lack of clarity in the docs. Reading them makes `ready` almost sound like it is to be avoided:

>This means that an element’s light DOM children may be initialized before or after the parent element, and an element’s siblings may become ready in any order.